src/core.js

```diff
@@ -9,21 +9,19 @@ const axios = require('axios');
 const log = require('./logger');
 const envs = require('./envs');
 
-if (!process.env.GITHUB_TOKEN) throw new Error('GITHUB_TOKEN must be defined in environment. Create one at https://github.com/settings/tokens');
-if (!process.env.GITHUB_WEBHOOK_SECRET) throw new Error('GITHUB_WEBHOOK_SECRET must be defined in environment. Create one at https://github.com/{OWNERNAME}/{REPONAME}/settings/hooks (swap in the path to your repo)');
-if (!process.env.ZEIT_API_TOKEN) throw new Error('ZEIT_API_TOKEN must be defined in environment. Create one at https://zeit.co/account/tokens');
+const env = process.env;
```

Comment:
We could probably take this one step further and just destructure the envars out right here:

```js
const {
  GITHUB_TOKEN, 
  GITHUB_WEBHOOK_SECRET, 
  GITLAB_TOKEN,
  GITLAB_WEBHOOK_SECRET,
  ZEIT_API_TOKEN
} = process.env;
```

what do you think?